packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -140,3 +147,52 @@ export class CustomScrollbar extends Component<Props> {
 }
 
 export default CustomScrollbar;
+
+const getStyles = (scrollbarStickLeft?: boolean): StylesInterface => {
+  return {
+    customScrollbar: css`
+      display: flex;
+      flex-grow: 1;
+      &:hover {
+        opacity: 0.8;
+        transition: opacity 0.3s ease-in-out;
+      }
+    `,
+    trackVertical: css`
+      border-radius: 3px;
+      width: 8px !important;
+      right: ${scrollbarStickLeft ? 0 : 2}px;
+      bottom: 2px;
+    `,
+    trackHorizontal: css`
+      border-radius: 3px;
+      height: 8px !important;
+      right: 2px;
+      bottom: 2px;
+      left: 2px;
+    `,
+    thumbVertical: css`
+      @include gradient-vertical($scrollbarBackground, $scrollbarBackground2);
```

Comment:
Do I need to convert this to CSS equivalent? @torkelo @hugohaggmark

Reply:
yes